functions/Remove-DbaBackup.ps1

```diff
@@ -148,25 +120,24 @@ It will also remove any backup folders that no longer contain backup files.
 			#>
 
 			[int]$Length = ($UserFriendlyRetention).Length
-			$Value = ($UserFriendlyRetention).Substring(0,$Length-1)
-			$Units = ($UserFriendlyRetention).Substring($Length-1,1)
+			$Value = ($UserFriendlyRetention).Substring(0, $Length - 1)
+			$Units = ($UserFriendlyRetention).Substring($Length - 1, 1)
 
 			# Validate that $Units is an accepted unit of measure
-			if ( $Units -notin @('h','d','w','m') ) {
+			if ( $Units -notin @('h', 'd', 'w', 'm') ) {
 				throw "RetentionPeriod '$UserFriendlyRetention' units invalid! See Get-Help for correct formatting and examples."
 			}
 
 			# Validate that $Value is an INT
-			if ( ![int]::TryParse($Value,[ref]"") ) {
+			if ( ![int]::TryParse($Value, [ref]"") ) {
 				throw "RetentionPeriod '$UserFriendlyRetention' format invalid! See Get-Help for correct formatting and examples."
 			}
 
-			switch ($Units)
-			{
-				'h' { $UnitString = 'Hours'; [datetime]$ReturnDatetime = (Get-Date).AddHours(-$Value)  }
-				'd' { $UnitString = 'Days';  [datetime]$ReturnDatetime = (Get-Date).AddDays(-$Value)   }
-				'w' { $UnitString = 'Weeks'; [datetime]$ReturnDatetime = (Get-Date).AddDays(-$Value*7) }
-				'm' { $UnitString = 'Months';[datetime]$ReturnDatetime = (Get-Date).AddMonths(-$Value) }
+			switch ($Units) {
+				'h' { $UnitString = 'Hours'; [datetime]$ReturnDatetime = (Get-Date).AddHours( - $Value)  }
```

Comment:
Can you revert this back to how it was and disable your editor from doing this?

Reply:
I understand it may not have adverse effect on intent of code and the execution, but `AddHours(-$value) looks better than `AddHours( - $value)` to me.